inst/rmarkdown/templates/nhsr-presentation/skeleton/skeleton.Rmd

```diff
@@ -1,9 +1,25 @@
 ---
-title: "Presentation"
-author: "[author](mailto:your@email.com)"
-institute: "[organisation](https://www.nhs.uk/)"
+title: "Presentation title"
+subtitle: "Presentation subtitle"
+author: "Author Name"
+institute: "NHS Organisation"
+date: "Presentation Date"
 output:
-  nhsrtheme::su_presentation: default
+  xaringan::moon_reader:
```

Comment:
this will need to be replaced by nhsrtheme::nhsr_presentation once I've done building that function